doc/snacks-dashboard.txt

```diff
@@ -12,8 +12,8 @@ Table of Contents                         *snacks-dashboard-table-of-contents*
 4. Types                                              |snacks-dashboard-types|
 5. Module                                            |snacks-dashboard-module|
   - Snacks.dashboard()            |snacks-dashboard-module-snacks.dashboard()|
+  - Snacks.dashboard.file_icon()|snacks-dashboard-module-snacks.dashboard.file_icon()|
   - Snacks.dashboard.have_pugin()|snacks-dashboard-module-snacks.dashboard.have_pugin()|
```

Comment:
Sorry again. `have_plugin` instead? Here and the other files. `dashboard.lua` also has it misspelled not in this diff, but I looked directly at the file to make sure.